compiler/rustc_span/src/def_id.rs

```diff
@@ -224,7 +224,7 @@ impl<D: Decoder> Decodable<D> for DefIndex {
 /// index and a def index.
 ///
 /// You can create a `DefId` from a `LocalDefId` using `local_def_id.to_def_id()`.
-#[derive(Clone, PartialEq, Eq, Copy)]
+#[derive(Clone, PartialEq, Eq, Copy, TypeFoldable, TypeVisitable)]
```

Comment:
why is that needed?

Reply:
Replaced with named structs in 1eda0645a07d1adc94eb9c25b9bf5d110ae31639